### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by richtier**
I removed the js tests because:
 - they were not run on ci
 - they were really just testing that objects existed on the page - not that they behave how we wanted

given the overhead of maintaining those dependencies, it's not worth keeping them.

fortunately, we have very little js to speak of.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,15 +10,11 @@
...
-    "gulp": "^3.9.1",
-    "gulp-sass": "^3.1.0",
-    "gulp-sourcemaps": "^2.6.1",
-    "karma": "^2.0.0",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-mocha": "^1.3.0",
-    "mocha": "^4.1.0",
-    "del": "^3.0.0"
+    "chai": "^4.2.0",
```
Comment (eadpearce) [inline package.json]: still keeping chai?

**Comment by eadpearce** [comment]
you can get rid of the test gulp task as well https://github.com/uktrade/great-domestic-ui/pull/52/files#diff-b9e12334e9eafd8341a6107dd98510c9R16

File: package.json (hunk 1 above)
Comment (richtier) [reply package.json]: oopw

File: README.md [hunk 2]
```diff
@@ -22,23 +21,23 @@ We aim to follow [GDS service standards](https://www.gov.uk/service-manual/servi
...
 
 ### SASS
 We use SASS CSS pre-compiler. If you're doing front-end work your local machine will also need the following dependencies:
 
 [node](https://nodejs.org/en/download/)
-
 [SASS](https://rubygems.org/gems/sass/versions/3.4.22)
 
+
 ### Run JS tests with gulp or npm
```
Comment (eadpearce) [inline README.md]: missed this before. you can also delete this bit

**Pull request merged**